setup.py

```diff
@@ -60,6 +60,7 @@
             "requests >= 2.1.0",
             "requests-oauthlib >= 0.4",
             "future >= 0.9.0",
+            "python-dateutil",
```

Comment:
Agreed.

@emattiza take a look at some of the other blog importers, you'll see they just wrap the import of the optional dependency with `try/catch ImportError` and if it's missing they raise an appropriate error message.